priv/repo/migrations/20170719103736_add_max_volunteer_per_task.exs

```diff
@@ -0,0 +1,10 @@
+defmodule CoursePlanner.Repo.Migrations.AddMaxVolunteerPerTask do
+  use Ecto.Migration
+
+  def change do
+    alter table(:tasks) do
+      remove :user_id
+      add :max_volunteers, :integer
```

Comment:
you added this as a required field in the ecto schema, we should then add `not null` to this migration as well, and in order to not break the migration in the staging server we also should have a default value (1 is fine to match the previous behavior)